custom_background/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# See LICENSE file for full copyright and licensing details.
+{
+    "name": "Custom Background",
+    "version": "17.1",
```

Comment:
"version": "17.0.0.0.1",

Reply:
not supported like this in master, correct me if I am doing any mistake because i have tried.